src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.domain;
+
+public class Car {
+    private Name name;
+    private Position position;
+
+    public Car(String name) {
+        this.name = new Name(name);
+        this.position = new Position(0);
```

Comment:
```suggestion
    this(new Name(name), new Position(0));
```
주 생성자 / 부 생성자를 정하여 부 생성자에서는 주 생성자를 호출하도록 구현 해보세요 !
그에 따른 장점이 어떤것이 있을지도 고민해보면 좋을것 같습니다 😄